datapackage_pipelines_mojp/bagnowka/processors/convert.py

```diff
@@ -15,25 +15,41 @@ def _get_schema(self):
     def _filter_resource(self, resource_descriptor, resource_data):
         for cm_row in resource_data:
             dbs_row = self._cm_row_to_dbs_row(cm_row)
+            logging.info("hello CONVERT")
```

Comment:
please remove, bear in mind that everything you log here appears in the pipeline dashboard